scss/components/_messages.scss

```diff
@@ -33,6 +33,20 @@
   @include u-background-image('i-info--neutral', 50% 2rem);
 }
 
+.message--success {
+  border-color: $primary-contrast;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets